odoo/http.py

```diff
@@ -308,7 +308,10 @@ def redirect(self, location, code=303, local=True):
         if local:
             location = urls.url_parse(location).replace(scheme='', netloc='').to_url()
         if request and request.db:
-            return request.registry['ir.http']._redirect(location, code)
+            try:
+                return request.registry['ir.http']._redirect(location, code)
+            except psycopg2.OperationalError:  # this is a quick fix waiting for https://github.com/odoo/odoo/pull/54102
+                pass
```

Comment:
Create a database, connect to it, drop in the database manager: an error appears saying that the db does not exist.
This fix is a quick solution to have green tests, but not checking the drop response could be another solution.